### ISSUE TITLE
Suppress warning "WebMock warning: ignoring query params in RFC 6570 template and checking them with WebMock"

### ISSUE THREAD

**Issue by giovannibenussi** [bblimke/webmock#726]
I'm using the instructions in the readme and I don't know why I'm getting this warning:
`WebMock warning: ignoring query params in RFC 6570 template and checking them with WebMock`

The code simplified is like this:
```
template = Addressable::Template.new 'https://example.com/{id}'
stub_request(:get, template).to_return(status: 200, body: '')
```

Having this warning in a large test suite is annoying because it prints out the warning too many times.

There is a way to supress this warning?

**Comment by bblimke** [comment]
@giovannibenussi Thanks for reporting.

I have just released version 2.3.5 which displays the warning only once.

If you want to remove the warning completely, you can set `RUBYOPT=W0` when running tests.

**Comment by giovannibenussi** [comment]
Sorry for the delay.. thanks a lot @bblimke! This will be really useful :-)
